config/powersync.yaml

```diff
@@ -0,0 +1,86 @@
+# powersync.yaml
+# Very similar to the config in the PowerSync dashboard, but a little more low-level
+replication:
+  connections:
+    - type: postgresql
+      sslmode: disable # okay for local/private network, not for public network
+
+      # The PowerSync server container can access the Postgres DB via the DB's service name.
+      # In this case the hostname is pg-db
+
+      # The connection URI or individual parameters can be specified.
+      # Individual params take precedence over URI params
+      uri: ${PS_PG_URI}
+
+      # Expose "execute-sql" for this connection
```

Comment:
Should we remove this for now since it's only useful for the CLI later? (if I understood Ralf correctly)

Reply:
Removed this now